src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -1106,6 +1109,14 @@ public unsafe void Dispose()
             Disposing();
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This isn't required. We use this on small methods that we need to be high perfomant. 
The first is true, but the second isn't.

Reply:
does it hurt us though?

Reply:
Yes, because it gives the wrong impression.
Basically, the rule is that unless you have looked at the ASM / have benchmark that show it, avoid it.